tests/test-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+## split tests
+test_suite_count=$(bazel query 'kind("test_suite", //tests/ios/unit-test:SplitTests)' | wc -l | xargs)
+if [[ "$test_suite_count" != "1" ]]; then
+  echo "Expected 1 test suite for split tests got $test_suite_count"
+  exit 1
+fi
+
+tests_count=$(bazel query 'tests(kind("test_suite", //tests/ios/unit-test:SplitTests))' | wc -l | xargs)
```

Comment:
`tests/xcodeproj-tests.sh` invokes `bazelisk`. Should we do the same here just for consistency?

Reply:
Updated